### PR TITLE
Add New fields to systems schema

### PR DESCRIPTION
**Pull request by patilsuraj767**

### CODE REVIEW

File: ros/lib/models.py [hunk 1]
```diff
@@ -34,6 +34,11 @@ class System(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     account_id = db.Column(db.Integer)
     inventory_id = db.Column(UUID(as_uuid=True), nullable=False)
+    display_name = db.Column(db.String(100))
```
Comment (apuntamb) [inline ros/lib/models.py:37]: do we want the display name to be unique?

Comment (patilsuraj767) [reply ros/lib/models.py:37]: No, it may happen that two user account has the system with the same display name. So it cannot be unique.

**Comment by apuntamb** [comment]
I guess the PR is already clean to get in. Just one query: do we also need to store Idling Time and IO wait fields in the `systems` table (fields shown in [mockup](https://marvelapp.com/prototype/9408d9j/screen/74816603))?

**Comment by patilsuraj767** [comment]
> I guess the PR is already clean to get in. Just one query: do we also need to store Idling Time and IO wait fields in the `systems` table (fields shown in [mockup](https://marvelapp.com/prototype/9408d9j/screen/74816603))?

We are not getting the idling time and IO wait from inventory, I think we will need to find a way to populate it through PCP metric. So I think the better place to store would be the performance profile table under performance record.

**Review by apuntamb — APPROVED** [review]
LGTM :+1:

**Comment by apuntamb** [comment]
Clean fix.
Merged, thanks @patilsuraj767!

**Pull request merged**